mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/CheckCmd.scala

```diff
@@ -45,4 +45,8 @@ class CheckCmd extends Command(name = "check",
     opt[String](name = "filter", default = "",
       description = "A sequence of regexes over transition numbers\n" +
         "to filter transitions at every step, e.g., (0|1),(1|2),4")
+  var tlc: Boolean = opt[Boolean](
```

Comment:
why don't we produce counterexamples in all supported formats: TLC's, TLA+, and JSON? Imagine that I have been running the tool for 1 day, just to realize that I forgot to specify `--tlc`. Dumping the counterexamples is cheap in comparison to model checking.

Reply:
yes, that's a nice suggestion -- and also removes some code, which is good;)